tests/new_upgrades/conftest.py

```diff
@@ -60,10 +61,30 @@ def shared_checkout(shared_name):
     ) as sat_checkout:
         sat_checkout.ready()
         sat_instance = bx_inst.from_inventory(
-            filter=f'@inv._broker_args.upgrade_group == "{shared_name}_shared_checkout"'
-        )[0]
-        sat_instance.setup()
-    return sat_instance
+            filter=f'@inv._broker_args.upgrade_group == "{shared_name}_shared_checkout" |'
+            '@inv._broker_args.workflow == "deploy-satellite"'
```

Comment:
Why we hardcode `deploy-satellite` workflow when we have already setting for that ? 

Why we use non upgrade deploy WF when these are upgrades ? 
I'd expect `deploy-satellite-upgrade` workflow instead

Reply:
As we discussed offline, the current upgrade scenarios refactor is not intended to catch issues that only occur when a Satellite has been upgraded multiple times, so we will continue to use the `deploy_satellite` and `deploy_capsule` workflows in these tests.

Good point about the setting for the workflow name, though, I will make that change.